script.js

```diff
@@ -1,4 +1,310 @@
+// What is the input going to be? Stone or Paper or Scissors
+// What should the output be? You played {input} and the computer played {stone/paper/scissors} so you {won/lost/drew}
+// How will the computer's option be randomly generated? Using math.random to generate a number between 0-2
+// How many different cases are there? 3 (win, lose, draw)
+
+var userWins = 0;
+var compWins = 0;
```

Comment:
Would suggest to spell out variables in full instead of shorthand in case the reader of your code is unfamiliar with the shorthand that you use.